programs/nft-voter/src/instructions/cast_nft_vote.rs

```diff
@@ -77,6 +80,9 @@ pub fn cast_nft_vote<'a, 'b, 'c, 'info>(
             NftVoterError::NftAlreadyVoted
         );
 
+        // Note: proposal.governing_token_mint must match voter_weight_record.governing_token_mint
+        // We don't verify it here because spl-gov does the check in cast_vote
+        // and it would reject voter_weight_record if governing_token_mint doesn't match
```

Comment:
nit: In an NFT vote, the governing token mint is any of the NFT mints, correct? If so, can we clarify the language to say something like: "it would reject voter_weight_record if nft_mint is not part of the governance"?

Reply:
No, it's incorrect. Once we enable the NFT plugin then the `governing_token_mint` is used only as identity of the voting population (community or council) and the tokens of that mint are not longer used for governance